pkgs/applications/networking/instant-messengers/telegram/cutegram/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub
+, qt5Libs, qtbase, qtquick1, qtmultimedia, qtquickcontrols, qtgraphicaleffects, makeQtWrapper
+, telegram-qml, libqtelegram-aseman-edition }:
+
+stdenv.mkDerivation rec {
+  name = "cutegram-v${version}";
```

Comment:
Yes, @ttuegel is right and I missed it. You have to remove `v` completely. And name has to be `cutegram-${version}` and version has to be `2.7.0-stable`.